Readme.md

```diff
@@ -246,6 +246,7 @@ TIPS
  - [Distributed parallel test (e.g. Travis Support)](https://github.com/grosser/parallel_tests/wiki/Distributed-Parallel-Tests-and-Travis-Support)
  - Debug errors that only happen with multiple files using `--verbose` and [cleanser](https://github.com/grosser/cleanser)
  - Shell alias: `alias prspec='parallel_rspec -m 2 --'`
+ - [Spring] to use spring you have to [patch it](https://github.com/grosser/parallel_tests/wiki) 
```

Comment:
Users will most likely have to reset redis + memcached + solr etc ... so
I'm afraid the solution won't work for everyone ... so something like

require 'parallel_tests/spring'
Parallel::Spring.reconnect do
  ActiveRecord::Base.reconnect!
end

might be a better approach ... but that's for another PR

On Fri, Dec 4, 2015 at 6:14 AM, betesh <notifications@github.com> wrote:

> In Readme.md
> <https://github.com/grosser/parallel_tests/pull/469#discussion_r46685247>:
>
> > @@ -246,6 +246,7 @@ TIPS
> >   - [Distributed parallel test (e.g. Travis Support)](https://github.com/grosser/parallel_tests/wiki/Distributed-Parallel-Tests-and-Travis-Support)
> >   - Debug errors that only happen with multiple files using `--verbose` and [cleanser](https://github.com/grosser/cleanser)
> >   - Shell alias: `alias prspec='parallel_rspec -m 2 --'`
> > + - [Spring] to use spring you have to [patch it](https://github.com/grosser/parallel_tests/wiki)
>
> Another suggestion: Instead of maintaining the code on the wiki, put it in
> the code base, and require "parallel_tests/spring_config" if
> defined?(::Spring)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel_tests/pull/469/files#r46685247>.
>